### PR TITLE
Adds a black turtleneck

### PR DESCRIPTION
**Pull request by Aikakakah**
## About the PR
Added a sleeveless black turtleneck.

## Why / Balance
Scarves are out, turtlenecks are in.

## Technical details
Added texture files, added to theatre drobe, added to DeltaV jumpsuit file.

## Media

image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: Added a black turtleneck!

### CODE REVIEW

**Comment by deltanedas** [comment]
why does it look so much like a web vest :trollface:

**Comment by Aikakakah** [comment]
> why does it look so much like a web vest :trollface:

Does it? 🫢 Might be the lighting, but I based it off the CE turtleneck and the MNK sleeveless outfit.

File: Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml [hunk 1]
```diff
@@ -349,3 +349,13 @@
   - type: Clothing
     sprite: DeltaV/Clothing/Uniforms/Jumpsuit/prosecutorred.rsi
 
+- type: entity
+  parent: ClothingUniformBase
+  id: ClothingUniformBlackTurtleneck
+  name: black turtleneck
+  description: A simple black turtleneck with the sleeves cut off.
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml]: this should use the rmc thing for taking off sleeves and be sleeved by default

**Review by deltanedas — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml (hunk 1 above)
Comment (Aikakakah) [reply Resources/Prototypes/DeltaV/Entities/Clothing/Uniforms/jumpsuits.yml]: Alright, hoping I didn't break anything by merging from the master branch, but it should be good.

**Comment by deltanedas** [comment]
sprite doesnt change when unrolled :trollface:

**Comment by Aikakakah** [comment]
😿 I'm gonna get this right eventually!

File: Resources/Textures/DeltaV/Clothing/Uniforms/Jumpsuit/black_turtleneck.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+  "version": 1,
+  "license": "CC-BY-SA-4.0",
+  "copyright": "Aikakakah",
```
Comment (deltanedas) [inline Resources/Textures/DeltaV/Clothing/Uniforms/Jumpsuit/black_turtleneck.rsi/meta.json]: make it more descriptive e.g. `Created by Aikakakah (github) for Delta-V`

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**